scanner/services/indexer.go

```diff
@@ -137,6 +147,7 @@ func (s *indexerService) GetOrCreateIndexReport(ctx context.Context, req *v4.Get
 		return nil, err
 	}
 
+	zlog.Info(ctx).Msg("creating index report for container image")
```

Comment:
Underneath `CreateIndexReport()` we already have a "creating index report for container image" log entry, with additional fields. Can you review these two and align them to keep the logs cleaner and less repetitive?